### PR TITLE
Document plugin dir in version string

### LINKED ISSUE #10467: [QA] unexpected debug output
### Pre-submission Checks

- [X] I [checked for similar issues](https://github.com/owncloud/client/issues?q=), but could not find any. I also checked the closed issues. I could not contribute additional information to any existing issue.
- [X] I will take the time to fill in all the required fields. I know that the bug report may be dismissed otherwise due to lack of information.

### Describe the QA issue

Seen with client 3.1.0 on linux.
When called from the command line, additional output is seen:
```
$ damkencloud -v
gui.platform: adding plugin directory "/opt/ownCloud/damkencloud/bin/../lib64/plugins"
damkencloud damkencloud 3.1.0.9987
e99d0f
Libraries Qt 5.15.2, OpenSSL 1.1.1l  24 Aug 2021 SUSE release 150400.7.19.1
Using virtual files plugin: suffix
opensuse-leap-5.14.21-150400.24.41-default

$ BtR -v
gui.platform: adding plugin directory "/opt/ownCloud/btr/bin/../lib64/plugins"
libpng warning: iCCP: CRC error
btr btr 3.1.0.9951
https://github.com/owncloud/client/commit/e99d0f90375fc3801d38d6761cbb319ea714ddb0
Libraries Qt 5.15.2, OpenSSL 1.1.1l  24 Aug 2021 SUSE release 150400.7.19.1
Using virtual files plugin: suffix
opensuse-leap-5.14.21-150400.24.41-default
suse@localhost:~> 
```

The line "gui.platform: adding plugin directory ..." should not be there.
The line "libpng warning: iCCP: CRC error" should not be there.

Expected behaviour:
- debug output is only printed, when called with --debug.

### Steps to reproduce the issue

.

### Screenshots

.

### Expected behavior

.

### Actual behavior

.

### PR DESCRIPTION
**Pull request by fmoc**
Fixes #10467.

### CODE REVIEW

File: src/libsync/theme.cpp [hunk 1]
```diff
@@ -414,10 +414,19 @@ QString Theme::aboutVersions(Theme::VersionFormat format) const
             gitUrl = gitSHA1(format) + br;
         }
     }
+
+#if defined(OC_PLUGIN_DIR)
+    const QString pluginDirPath = QDir(QCoreApplication::applicationDirPath()).filePath(QStringLiteral(OC_PLUGIN_DIR));
```
Comment (TheOneRing) [inline src/libsync/theme.cpp:419]: You have to agree that this looks like the wrong place to modify addLibraryPath

**Review by TheOneRing — CHANGES_REQUESTED** [review]

File: src/libsync/theme.cpp [hunk 2]
```diff
@@ -414,10 +414,19 @@ QString Theme::aboutVersions(Theme::VersionFormat format) const
             gitUrl = gitSHA1(format) + br;
         }
     }
+
+
+#if defined(OC_PLUGIN_DIR)
+    const QString pluginDirPath = QDir(QCoreApplication::applicationDirPath()).filePath(QStringLiteral(OC_PLUGIN_DIR));
+#else
+    const QString pluginDirPath = QStringLiteral("not set");
```
Comment (TheOneRing) [inline src/libsync/theme.cpp:422]: Can we just not mention it at all?
OC_PLUGIN_DIR is just a workaround for the weird package layout we provide on linux.

Comment (fmoc) [reply src/libsync/theme.cpp:422]: We intentionally introduced this in 4fbcf5cd. We should make the path available _somewhere_ at least.

Comment (fmoc) [reply src/libsync/theme.cpp]: You clarified you just want to remove the else branch. Sure, let's do that.

**Comment by TheOneRing** [comment]
Wrong branch

**Comment by ownclouders** [comment]
Results for <strong>GUI-tests-ocis</strong> https://drone.owncloud.com/owncloud/client/14242/7/1

<details>


</details>

**Comment by ownclouders** [comment]
Results for <strong>GUI-tests-oc10</strong> https://drone.owncloud.com/owncloud/client/14242/6/1

<details>


</details>

File: src/libsync/theme.cpp [hunk 3]
```diff
@@ -414,10 +414,19 @@ QString Theme::aboutVersions(Theme::VersionFormat format) const
             gitUrl = gitSHA1(format) + br;
         }
     }
+
+
+#if defined(OC_PLUGIN_DIR)
+    const QString pluginDirComment = QStringLiteral("Plugin dir: %1%2").arg(QDir(QCoreApplication::applicationDirPath()).filePath(QStringLiteral(OC_PLUGIN_DIR))).arg(br);
```
Comment (TheOneRing) [inline src/libsync/theme.cpp:420]: QCoreApplication::translate ?

File: src/libsync/theme.cpp [hunk 4]
```diff
@@ -414,10 +414,19 @@ QString Theme::aboutVersions(Theme::VersionFormat format) const
             gitUrl = gitSHA1(format) + br;
         }
     }
+
+
+#if defined(OC_PLUGIN_DIR)
+    const QString pluginDirComment = tr("ownCloudTheme::qtVer", "Plugin dir: %1%2").arg(QDir(QCoreApplication::applicationDirPath()).filePath(QStringLiteral(OC_PLUGIN_DIR)), br);
```
Comment (TheOneRing) [inline src/libsync/theme.cpp:420]:
```suggestion
    const QString pluginDirComment =  QCoreApplication::translate("ownCloudTheme::pluginDir", "Plugin dir: %1%2").arg(QDir(QCoreApplication::applicationDirPath()).filePath(QStringLiteral(OC_PLUGIN_DIR)), br);
```

**Review by TheOneRing — APPROVED** [review]

**Comment by saw-jan** [comment]
Tested in https://github.com/owncloud/client/issues/10467#issuecomment-1429099503

**Pull request merged**